client/me/security-checkup/index.jsx

```diff
@@ -51,4 +48,10 @@ module.exports = React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+SecurityCheckup.displayName = 'SecurityCheckup';
```

Comment:
**Note: I'm just thinking out loud here, feel free to ignore!** 🙂

Do we rely on .displayName? .name seems to be set successfully with the ES^ class extends syntax.
I'm also looking in to having babel take care of inferring the displayName as it does with React.createClass syntax ([reference](https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-react-display-name)).  
There have been [discussions](https://github.com/babel/babel/issues/1331) about this and it looks like at least .name is currently available... I'm not sure how involved it would be to fallback to this or if this even worth considering... I did a little test which looks like:

    // SecurityCheckup.displayName = 'SecurityCheckup'; // note: commented out

Logging these properties:

    console.log({
        displayName: this.constructor.displayName,
        name: this.constructor.name
    });

gives:

<img width="654" alt="screen shot 2016-10-24 at 23 27 01" src="https://cloud.githubusercontent.com/assets/4335450/19645977/28d2f4e0-9a43-11e6-8fc3-fb1083e7c6f6.png">